empress/support_files/js/empress.js

```diff
@@ -854,9 +853,31 @@ define([
     Empress.prototype.getNodeCoords = function () {
         var tree = this._tree;
         var coords = [];
+        var scope = this;
+        var visible = function (node) {
+            return scope.getNodeInfo(node, "visible");
+        };
+        var comp;
+        if (this.drawNodeCircles === 0) {
+            comp = function (node) {
+                return (
+                    visible(node) && !tree.isleaf(tree.postorderselect(node))
```

Comment:
This is pseudo-code but I think this would help further reduce the number of circles drawn on screen:

```suggestion
                    visible(node) && !tree.isleaf(tree.postorderselect(node)) && tree.children(...).length === 1```

Reply:
I added a forth option (show internal nodes with one descendant) and made it the default. Here is an example.
![Screenshot from 2021-02-17 15-54-45](https://user-images.githubusercontent.com/28127694/108283762-bd1f8980-7138-11eb-9c83-f6ed10f1e524.png)

Reply:
Excellent, that looks exactly like what we need! Just for completeness how does that clade in the tree look when you show all node circles?

Reply:
Here is the clade. The node highlighted in the above clade is located on the bottom branch.

![Screenshot from 2021-02-17 17-54-22](https://user-images.githubusercontent.com/28127694/108292605-3de68180-7149-11eb-9c04-443096e1a017.png)

Reply:
Perfect, this looks great! Whenever you get a chance feel free to push the code and I'll be happy to re-review the code.

Reply:
my bad. I had commited it but forgot to push it.